### PR TITLE
Add frequency knob to manipulate MIDI clock signals per beat

### PR DESCRIPTION
**Pull request by brushyfork23**

### CODE REVIEW

File: src/Synferno_15/Synferno_15.ino [hunk 1]
```diff
@@ -278,8 +281,8 @@ void showOffset(byte count) {
   }
 }
 
-void showFrequency(byte count) {
-  static byte lastCount = 255;
+void showFrequency(int step) {
```
Comment (brushyfork23) [inline src/Synferno_15/Synferno_15.ino:284]: int to allow negative values

**Comment by linenoise** [comment]
Would you mind if we merged this once @the-magister has been able to test his changes in the shop on prod hardware?

**Comment by the-magister** [comment]
Uh, so everyone should understand that we've arrived at the limit of what I
know about git. I gather that Michael did some stuff, and some action is
required to do something about that. Mike won't take that action, b/c git
was written by sadists for masochists.

**Comment by brushyfork23** [comment]
Lol.  It was too late last night, but I just now tested compilation (and fixed the errors I found).  It's safe to merge now.

**Comment by the-magister** [comment]
So, one of the practices that I've adopted is to keep the master branch at
"I know this works on the hardware because I've tested it", and branches
for updates from the safety of the couch. I'll also save a branch after
in-flight repairs like 'cnw-2017' that work in the field.

At this point it doesn't really matter. Later, if you introduce a change to
master without a hardware test, it can make a difference. Like "super crazy
fireball that was spectacularly dangerous" kind  of a difference.  Um,
full-disclosure--I've done that before and would spare you that experience.

This is something like the Test, Dev, Prod model in software deployment, so
I hope this is a familiar pattern?

**Comment by brushyfork23** [comment]
I'm totally into that.  Will not merge future branches until they've been
tested on hardware.

**Comment by the-magister** [comment]
I'm just happy to know what "merge" means in this context. I'm not a
software engineer; I just play one on TV.

**Pull request merged**